tabpy/tabpy_server/handlers/evaluation_plane_handler.py

```diff
@@ -72,7 +76,14 @@ def _post_impl(self):
         user_code = body["script"]
         arguments = None
         arguments_str = ""
-        if "data" in body:
+        if self.arrow_server is not None and "dataPath" in body:
+            # arrow flight scenario
+            print("arrow flight scenario")
+            arrow_data = self.get_arrow_data(body["dataPath"])
+            if arrow_data is not None:
+                arguments = {"_arg1": arrow_data}
+        elif "data" in body:
+            # backwarding
```

Comment:
Rather than "backwarding" can we call this something like "data in payload scenario"